album_cover_generator.py

```diff
@@ -0,0 +1,22 @@
+import io
+import PIL
```

Comment:
The relevant imports are explicit already, there is no need to import the whole library here

Reply:
Changed!